src/components/LightButton.vue

```diff
@@ -0,0 +1,11 @@
+<script setup lang="ts">
```

Comment:
Quick question, are we using typescript?

Reply:
ya